example/provider-local/seed2/.gitignore

```diff
@@ -0,0 +1 @@
+kubeconfig
```

Comment:
Can we rename this folder from `example/provider-local/seed2` to `example/provider-local/seed-kind2`?